substrings.rb

```diff
@@ -0,0 +1,12 @@
+def substrings(str, dictionary)
+  str.downcase!
+  counts = {}
+  dictionary.each do |word| 
+    words = str.scan(word) 
+      if words.length > 0
```

Comment:
I think you can move this block back two spaces

Reply:
good catch! :D